### DISCUSSION TITLE
Flask ready command

### DISCUSSION THREAD

**Discussion by ashupal86** [pallets/flask#5518] [Q&A] [answered]
is there any command to just type and create a basic flask app that show hello world or any thing like npm does?

**Comment by ThiefMaster** [comment]
https://github.com/pallets/flask?tab=readme-ov-file#a-simple-example

It's so simple, there's no need for a command to generate this...

**Reply by ashupal86** [reply]
I just want to know is there any flask command that create a basic flask project that displays hello world so I can just create a basic app in one line without starting from the scratch.

**Reply by CheeseCake87** [reply, marked as answer]
> I just want to know is there any flask command that create a basic flask project that displays hello world so I can just create a basic app in one line without starting from the scratch.

I wrote a Flask extension that pretty much does this. [Flask-Imp CLI](https://cheesecake87.github.io/flask-imp/v4/cli_commands-flask-imp_init.html) - The code for generating the files and folders is here: [CLI Code](https://github.com/CheeseCake87/flask-imp/tree/master/flask_imp/_cli) , feel free to adjust and use the code if you find it useful.
